website/docs/d/is_lb_listener_policy_rules.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "VPC infrastructure"
+page_title: "IBM : ibm_is_lb_listener_policy_rules"
+description: |-
+  Get information about LoadBalancerListenerPolicyRuleCollection
+---
+
+# ibm_is_lb_listener_policy_rules
+
+Provides a read-only data source for LoadBalancerListenerPolicyRuleCollection. For more information, about load balancer listener policy and rules, see [layer 7 load balancing policies and rules](https://cloud.ibm.com/docs/vpc?topic=vpc-layer-7-load-balancing).
+## Example Usage
+
+```terraform
+data "ibm_is_lb_listener_policy_rules" "example" {
+	listener = ibm_is_lb_listener.example.listener_id
+	lb = ibm_is_lb.example.id
+	policy = ibm_is_lb_listener_policy.example.policy_id
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+- `listener_id` - (Required, Forces new resource, String) The listener identifier.
```

Comment:
Correction in the name as per the actual parameters.

Reply:
Can you check again now please